src/sentry/http.py

```diff
@@ -272,7 +272,14 @@ def fetch_file(url, headers=None, domain_lock_enabled=True, outfile=None):
             if cl > settings.SENTRY_FETCH_MAX_SIZE:
                 raise OverflowError()
 
-            contents = []
+            try:
+                from cStringIO import StringIO
```

Comment:
Definitely not `six.StringIO`. `cStringIO` is a `BytesIO` which is what is needed here.